installation/language/de-DE/joomla.ini

```
@@ -285,7 +285,7 @@ WARNING="Warnung"
 JLIB_JS_AJAX_ERROR_CONNECTION_ABORT="Ein Verbindungsabbruch trat beim Abruf der JSON-Daten auf."
 JLIB_JS_AJAX_ERROR_NO_CONTENT="Kein Inhalt wurde zurückgegeben."
 JLIB_JS_AJAX_ERROR_OTHER="Beim Abrufen von JSON-Daten wurde ein HTTP-Statuscode %s zurückgegeben."
-JLIB_JS_AJAX_ERROR_PARSE="Ein Parsing-Fehler trat bei der Verarbeitung der folgenden JSON-Daten auf:<br><code style=\'color:inherit;white-space:pre-wrap;padding:0;margin:0;border:0;background:inherit;\'>%s</code>"
+JLIB_JS_AJAX_ERROR_PARSE="Ein Parsing-Fehler trat bei der Verarbeitung der folgenden JSON-Daten auf:<br><code style=\"color:inherit;white-space:pre-wrap;padding:0;margin:0;border:0;background:inherit;\">%s</code>"
```

```suggestion
JLIB_JS_AJAX_ERROR_PARSE="Ein Parsing-Fehler trat bei der Verarbeitung der folgenden JSON-Daten auf:<br><code style='color:inherit;white-space:pre-wrap;padding:0;margin:0;border:0;background:inherit;'>%s</code>"
```
Inline styles sind doof. Kommt aber wahrscheinlich auch schon aus der Quelle?